docs/framework/builder-basics.mdx

```diff
@@ -2,134 +2,178 @@
 title: "Builder basics"
 ---
 
-Framework Builder works as an overlay of the running app; you edit your app while it's running. It gives you an accurate representation of what the app will look like and how it'll behave, without the need to constantly preview it. Changes to the user interface are automatically saved into `.wf/` folders.
+Framework Builder’s interface is an overlay on top of the running app, which allows you to edit your app while it’s running. This approach gives you an accurate representation of how the app will look and behave without the need to constantly preview it. 
 
-## Modes
+## Framework Builder’s modes
```

Comment:
Ditto here, "Writer Framework Builder's modes"

Reply:
@samjulien Done — and not just for this instance, but all instances of “Framework” in the text and alt text.